assignments/favourite_movie/favourite_movie.js

```diff
@@ -2,6 +2,13 @@ const movies = [];
 
 function favouriteMovie(operation, movie) {
     // Write your code here
+    if(operation === "add"){
+        movies.push(movie);
+    }
+    else if(operation === "remove"){
```

Comment:
may be  your test cases pass but think if array is empty then it give error so edit else if 
else if(operation === "remove" && arraysize>=1 )
it will also work with more appropriate

Reply:
Hi, really appreciate your insights. It would be a wise decision to add the extra piece of code in order to handle the scenario.  I did checked running the case where i tried to use pop method on an empty array and its not throwing any error. I could be wrong though ( I frequently am wrong ), I would really like if you could please check the facts once more and correct me if I am wrong. Thank you.